FamilySearchIosSampleApp/LoginVC.swift

```diff
@@ -156,31 +149,38 @@ class LoginVC: UIViewController {
                 if let usersJsonObject = currentUserJson["users"] as? [[String : AnyObject]]
                 {
                     let user = User()
-                    let userJsonObject = usersJsonObject.first!
-                    
-                    user.id = userJsonObject["id"] as? String
-                    user.contactName = userJsonObject["contactName"] as? String
-                    user.helperAccessPin = userJsonObject["helperAccessPin"] as? String
-                    user.givenName = userJsonObject["givenName"] as? String
-                    user.familyName = userJsonObject["familyName"] as? String
-                    user.email = userJsonObject["email"] as? String
-                    user.country = userJsonObject["country"] as? String
-                    user.gender = userJsonObject["gender"] as? String
-                    user.birthDate = userJsonObject["birthDate"] as? String
-                    user.phoneNumber = userJsonObject["phoneNumber"] as? String
-                    user.mailingAddress = userJsonObject["mailingAddress"] as? String
-                    user.preferredLanguage = userJsonObject["preferredLanguage"] as? String
-                    user.displayName = userJsonObject["displayName"] as? String
-                    user.personId = userJsonObject["personId"] as? String
-                    user.treeUserId = userJsonObject["treeUserId"] as? String
-                    
-                    // The Memories activity will need the URL that comes from user.links.artifact.href
-                    // in order to get the memories data
-                    let links = userJsonObject["links"] as! NSDictionary
-                    let artifacts = links["artifacts"] as! NSDictionary
-                    user.artifactsHref = artifacts["href"] as? String
+                    guard usersJsonObject.first == nil else
```

Comment:
This use of `guard` is kind of strange.  You nil checking the `usersJsonObject` but it would read cleaner if you did:
```
if let userJsonObject = usersJsonObject.first {
    //Extract your data here
}
```